ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -51,10 +51,12 @@ Item {
             }
             property string categoryName: ""
             property bool categoryOpened: true
+            property bool categoryHasItems: false
 
             function updateCategoryData(catId) {
                 categoryName = root.model.sourceModel.getCategoryNameForCategoryId(catId)
                 categoryOpened = root.model.sourceModel.getCategoryOpenedForCategoryId(catId)
+                categoryHasItems = root.model.sourceModel.getCategoryHasItemsForCategoryId(catId)
```

Comment:
I think you forgot to push a commit that adds this function to the model

Reply:
Indeed!